app/javascript/modules/listingDetails/ListingDetailsEligibility.tsx

```diff
@@ -20,6 +22,28 @@ export const ListingDetailsEligibility = ({
   listing,
   imageSrc,
 }: ListingDetailsEligibilityProps) => {
+  /* TODO: Implement updated API to get actual data */
+  const HMITableHeaders = {
+    householdSize: "t.householdSize",
+    maxIncomeMonth: "t.maximumIncomeMonth",
+    maxIncomeYear: "t.maximumIncomeYear",
+  }
+  const HMITableData = [
+    {
+      householdSize: { content: "1 person" },
+      maxIncomeMonth: { content: "$1,111 per month" },
+      maxIncomeYear: { content: "$51,111 per year" },
```

Comment:
can we add more mock data so we can see the zebra striping and 'show more' link?

also, let's translate the content